src/honeysql/types.cljc

```diff
@@ -0,0 +1,84 @@
+(ns honeysql.types
```

Comment:
It's really hard to see what changed between these two--anyway to re-commit these so the diff shows up correctly?